package.json

```diff
@@ -19,18 +19,26 @@
     "repository": "git://github.com/fluid-project/infusion.git",
     "main": "./src/module/fluid.js",
     "scripts": {
-        "prepare": "npm run loadDependencies && npm run buildSass",
-        "prepublishOnly": "npm run buildDists && npm run test:browserBundles",
-        "pretest": "node node_modules/rimraf/bin.js reports/* coverage/*",
-        "test": "npm run test:browser && npm run test:node",
-        "test:browser": "node node_modules/testem/testem.js ci --file tests/testem.js",
-        "test:browserBundles": "node node_modules/testem/testem.js ci --file tests/bundle-tests/testem.js",
-        "test:node": "node node_modules/nyc/bin/nyc.js node tests/node-tests/basic-node-tests.js",
-        "posttest": "node node_modules/nyc/bin/nyc.js report -r text-summary -r html -r json",
-        "buildDists": "grunt buildDists",
-        "buildSass": "grunt buildSass",
-        "loadDependencies": "grunt loadDependencies",
-        "lint": "grunt lint"
+        "prepare": "run-s deps build:sass",
+        "prepublishOnly": "npm run test:bundles",
+        "pretest": "rimraf reports/* coverage/*",
+        "test": "run-s test:browser test:node",
+        "test:browser": "testem ci --file tests/testem.js",
+        "test:bundles": "run-s test:bundles:*",
+        "test:bundles:prepare": "npm run build:dists",
+        "test:bundles:run": "testem ci --file tests/bundle-tests/testem.js",
+        "test:node": "nyc node tests/node-tests/basic-node-tests.js",
+        "test:report": "nyc report -r text-summary -r html -r json",
+        "posttest": "npm run test:report",
+        "build:all": "grunt",
```

Comment:
This is a great set of improvements, thanks. If this is the new preferred way of triggering builds, we should update the README section named "Package Types" to account for it. I think it would also make sense to add a new README section describing some of the other scripts, e.g. "deps"